architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -175,29 +175,158 @@ endif::[]
+If the source image has changed, that change is picked up and reflected in the image stream, which trigger the Build and/or Deployment flow, depending upon the Build or Deployment configuration.
+
+* You can share images using fine-grained access control and quickly distribute images across your teams.
+
+* If the source image changes, the image stream tag will still point to a known-good version of the image, ensuring that your application will not break unexpectedly.
+
+* You can xref:../../admin_guide/manage_authorization_policy.adoc#admin-guide-manage-authorization-policy[configure security] around who can view and use the images through permissions on the image stream objects. 
+
+* Users that lack permission to read or list images on the cluster level can still retrieve the images tagged in a project using image streams.
+
+
+For a curated set of image streams, see the
 link:https://github.com/openshift/library[OpenShift Image Streams and Templates
 library].
 
-For example, if a deployment is using a certain image and a new version of that
-image is created, a deployment could be automatically performed.
+When using image streams, it is important to understand what the image stream tag is pointing to and how changes to tags and images can affect you. For example:
+
+* If your image stream tag points to a Docker image tag, you need to understand how that Docker image tag is updated. For example, a Docker image tag `docker.io/ruby:2.4` will probably always point to a v2.4 ruby image. But, a Docker image tag `docker.io/ruby:latest` will probably change with major versions. So, the Docker image tag that a image stream tag points to can tell you how stable the image stream tag will be, if you choose to reference it.
+
+* If your image stream tag follows another image stream tag (it does not point directly to a docker image tag), it is possible that the image stream tag will be updated to follow a different image stream tag in the future. Again, this could result in picking up an incompatible version change.
+
+=== Important terms
+
+Docker repository::
+A collection of related docker images and tags identifying them.
+For example, the OpenShift Jenkins images are in a Docker repository:
++
+----
+docker.io/openshift/jenkins-2-centos7
+----
+
+Docker registry::
+A content server that can store and service images from Docker repositories. 
+For example:
++
+----
+registry.access.redhat.com
+----
+
+Docker image::
+ A specific set of content that can be run as a container. Usually associated with a particular tag within a docker repository.
+
+Docker image tag::
+A label applied to a Docker image in a repository that distinguishes a specific image. For example, here *3.6.0* is a tag:
+
+----
+docker.io/openshift/jenkins-2-centos7:3.6.0
+----
+
+[NOTE]
+====
+A Docker image tag can be updated to point to new Docker image content at any time.
+====
+
+Docker image ID::
+A SHA code that can be used to pull an image. For example:
```

Comment:
Since this is the first reference to SHA in this doc, do you think it needs to be defined? If it's common knowledge, it could slide.